test/include/util/transaction_test_util.h

```diff
@@ -242,6 +242,11 @@ class LargeTransactionTestObject {
    */
   transaction::TransactionManager *GetTxnManager() { return &txn_manager_; }
 
+  /**
+   * @return the data table used by this test
+   */
+  storage::DataTable *GetTable() { return &table_; }
```

Comment:
You don't appear to use this, and you should not use this even in testing.  If there is something at the DataTable level that you need to interact with, even if it is for testing, then there is an API problem with SqlTable.  Recommend removing.